platform-includes/sourcemaps/overview/javascript.capacitor.mdx

```diff
@@ -0,0 +1,48 @@
+## Uploading Source Maps
+
+The easiest way to configure uploading source maps is by using the Sentry Wizard:
+
+<Include name="sourcemaps-wizard-instructions.mdx" />
+
+After choosing your Sentry project, the setup process will prompt you to select a framework. This step is crucial for configuring Sentry correctly. Make sure to choose the appropriate framework based on your project’s requirements.
+
+```example
+◆  Which framework, bundler or build tool are you using?
+│  ● Angular
```

Comment:
I'm not sure why some frameworks have a solid circle (angular) vs an open one (next.js)? It's confusing because the solid ones feel like it's selected